src/components/Pages/Collection/index.js

```diff
@@ -99,9 +100,10 @@ const Collection = ({ id, location }) => {
           return result.json()
         })
         .then(async (result) => {
-          Object.assign(imageGroups, mapImageDirectories(result.data.listImageGroupsForS3.items))
+          _.merge(imageGroups, mapImageDirectories(result.data.listImageGroupsForS3.items))
```

Comment:
I think you could do this without resorting to _.merge. We don't have super nested objects. You could just loop through the results and append to imageGroups, creating the top level key if it doesn't already exist.